app/communication/tests/test_banner_model.py

```diff
@@ -0,0 +1,48 @@
+from datetime import timedelta
+
+import pytest
+
+from app.communication.exceptions import (
+    AnotherVisibleBannerError,
+    DatesMixedError,
+)
+from app.communication.factories.banner_factory import BannerFactory
+from app.util.utils import now
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    "visible_from, visible_until, expected_value",
+    [
+        (now() + timedelta(hours=1), now() + timedelta(hours=2), False),
+        (now(), now() + timedelta(2), True),
+        (now() - timedelta(hours=1), now() + timedelta(hours=1), True),
+        (now() - timedelta(hours=2), now() - timedelta(hours=1), False),
+    ],
+)
+def test_banner_is_visible_with_different_dates(
+    visible_from, visible_until, expected_value
+):
+    "Banner is be visible when visible_from is passed and visible_until is not passed or None"
+    banner = BannerFactory(visible_from=visible_from, visible_until=visible_until)
+
+    is_visible = banner.is_visible
+
+    assert is_visible == expected_value
```

Comment:
Fin test, men siden `is_visible` ikke brukes for filtreringen i viewet (som er bra) så mener jeg at du kan fjerne `is_visible` fra modellen og heller lage en slik parametrisert integrasjonstest som sjekker at filtreringen du har lagt til i `visible`-action fungerer